specification/specs/common.ts

```diff
@@ -149,3 +149,6 @@ class LatLon {
   lat: double
   lon: double
 }
+
+/** For empty Class assignments */
+class EmptyObject {}
```

Comment:
cc @Mpdreamz @swallez is this ok for you or should we build something more advanced?